### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yaoshiang**
Removes all references to MFU. 

I think that it is very important that our repo always represents an accurate state of the repo and product. These numbers are applied science - the numbers must be meaningful to consumers. These MFU numbers used a memory efficient optimizer that is not proven to converge as well as AdamW.

### CODE REVIEW

**Review by tengyifei — COMMENTED** [review]
I get the sentiment but have questions:

- Right now https://github.com/AI-Hypercomputer/torchprime?tab=readme-ov-file#supported-models lists that these models are optimized and we publish performance. If we remove the published performance, what should we do in the model status table?

- Granted I don't have evidence, but the extra memory utilization from AdamW compared to AdaFactor is just one extra param. The perf will probably be quite similar once we fix https://github.com/AI-Hypercomputer/torchprime/issues/36.


Maybe instead we call out that the optimizer is Adafactor and we're working on AdamW numbers?

**Comment by yaoshiang** [comment]
> I get the sentiment but have questions:
> 
> * Right now https://github.com/AI-Hypercomputer/torchprime?tab=readme-ov-file#supported-models lists that these models are optimized and we publish performance. If we remove the published performance, what should we do in the model status table?
> * Granted I don't have evidence, but the extra memory utilization from AdamW compared to AdaFactor is just one extra param. The perf will probably be quite similar once we fix [[torch_xla] Training loss is NaN under the AdamW optimizer #36](https://github.com/AI-Hypercomputer/torchprime/issues/36).
> 
> Maybe instead we call out that the optimizer is Adafactor and we're working on AdamW numbers?

I think there is some consensus that the ordering of "implemented, optimized, converges" should switch to "implemented, converges, optimized", and, that a model's not really complete until it has all three (plus documentation). Until then, it is work in progress, and best not to publish. That's a stronger statement than this PR, however, - we should discuss as a team.

Re: this PR, I don't think it makes sense to describe Adafactor for Llama because one of the main reference points was the Llama3 paper itself, which listed IIRC 38%-43% MFU with Adam. The memory usage of Adam is material, and the batch size is probable to go down.

I'd say I have a 25% confidence that that the MFU numbers will change materially under Adam - put another way, I only have a 75% confidence level that the number will hold up under what I think is the correct experimental setup... so that's not high enough confidence for me to publish. 

Totally understand if you aren't convinced... let's discuss as a team next week.

**Comment by tengyifei** [comment]
I think there's also a product factor i.e. if we already told someone about certain MFU numbers, would they come here and find that those are now gone vs moved to an appendix with a disclaimer that it's optimized with Adafactor for now. cc @cloudchrischan

**Comment by tengyifei** [comment]
I'll defer to @vlasenkoalexey or @cloudchrischan to approve the PR

**Comment by cloudchrischan** [comment]
Aligned with:

1. Removing reference MFU
2. Reverting all "Optimized" status indicators to a TODO state

**Comment by cloudchrischan** [comment]
@yaoshiang Can you revert the README status indicators to TODO in this PR?

**Review by cloudchrischan — APPROVED** [review]

**Comment by cloudchrischan** [comment]
Approved, @yaoshiang confirmed updating the table in another PR

**Review by qihqi — APPROVED** [review]

**Comment by vlasenkoalexey** [comment]
Agree that MFU probably should not be stated on the README page, especially if we have some doubts that numbers would hold if we change optimizer, which we most likely have to do. Also Yifei did non-trivial amount of work to achieve those results and it is not clear when we'll be able to achieve same results on AdamW. 

Therefore I'd suggest to move those numbers to another place, https://github.com/AI-Hypercomputer/torchprime?tab=readme-ov-file#supported-models could be a good one with asterisk that MFU is achieved using adafactor optimizer which is more memory efficient.

**Comment by yaoshiang** [comment]
> Agree that MFU probably should not be stated on the README page, especially if we have some doubts that numbers would hold if we change optimizer, which we most likely have to do. Also Yifei did non-trivial amount of work to achieve those results and it is not clear when we'll be able to achieve same results on AdamW.
> 
> Therefore I'd suggest to move those numbers to another place, https://github.com/AI-Hypercomputer/torchprime?tab=readme-ov-file#supported-models could be a good one with asterisk that MFU is achieved using adafactor optimizer which is more memory efficient.

I've added the numbers to a metrics tracking CSV including Adafactor as a column.
